<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OBSBasicFilters</class>
 <widget class="QDialog" name="OBSBasicFilters">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>861</width>
    <height>726</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Basic.Filters</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="sizeConstraint">
      <enum>QLayout::SetMinimumSize</enum>
     </property>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QWidget" name="asyncWidget" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="asyncLabel">
            <property name="text">
             <string>Basic.Filters.AsyncFilters</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="FocusList" name="asyncFilters">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="contextMenuPolicy">
             <enum>Qt::CustomContextMenu</enum>
            </property>
           </widget>
          </item>
          <item alignment="Qt::AlignLeft">
           <widget class="QWidget" name="widget" native="true">
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <property name="spacing">
              <number>4</number>
             </property>
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QPushButton" name="addAsyncFilter">
               <property name="maximumSize">
                <size>
                 <width>22</width>
                 <height>22</height>
                </size>
               </property>
               <property name="text">
                <string notr="true"/>
               </property>
               <property name="icon">
                <iconset resource="obs.qrc">
                 <normaloff>:/res/images/add.png</normaloff>:/res/images/add.png</iconset>
               </property>
               <property name="autoDefault">
                <bool>false</bool>
               </property>
               <property name="flat">
                <bool>true</bool>
               </property>
               <property name="themeID" stdset="0">
                <string>addIconSmall</string>
               </property>
               <property name="accessibleName">
                <string>Add</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="removeAsyncFilter">
               <property name="maximumSize">
                <size>
                 <width>22</width>
                 <height>22</height>
                </size>
               </property>
               <property name="text">
                <string notr="true"/>
               </property>
               <property name="icon">
                <iconset resource="obs.qrc">
                 <normaloff>:/res/images/list_remove.png</normaloff>:/res/images/list_remove.png</iconset>
               </property>
               <property name="autoDefault">
                <bool>false</bool>
               </property>
               <property name="flat">
                <bool>true</bool>
               </property>
               <property name="themeID" stdset="0">
                <string>removeIconSmall</string>
               </property>
               <property name="accessibleName">
                <string>Remove</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="moveAsyncFilterUp">
               <property name="maximumSize">
                <size>
                 <width>22</width>
                 <height>22</height>
                </size>
               </property>
               <property name="text">
                <string notr="true"/>
               </property>
               <property name="icon">
                <iconset resource="obs.qrc">
                 <normaloff>:/res/images/up.png</normaloff>:/res/images/up.png</iconset>
               </property>
               <property name="autoDefault">
                <bool>false</bool>
               </property>
               <property name="flat">
                <bool>true</bool>
               </property>
               <property name="themeID" stdset="0">
                <string>upArrowIconSmall</string>
               </property>
               <property name="accessibleName">
                <string>MoveUp</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="moveAsyncFilterDown">
               <property name="maximumSize">
                <size>
                 <width>22</width>
                 <height>22</height>
                </size>
               </property>
               <property name="text">
                <string notr="true"/>
               </property>
               <property name="icon">
                <iconset resource="obs.qrc">
                 <normaloff>:/res/images/down.png</normaloff>:/res/images/down.png</iconset>
               </property>
               <property name="autoDefault">
                <bool>false</bool>
               </property>
               <property name="flat">
                <bool>true</bool>
               </property>
               <property name="themeID" stdset="0">
                <string>downArrowIconSmall</string>
               </property>
               <property name="accessibleName">
                <string>MoveDown</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="Line" name="separatorLine">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="effectWidget" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Basic.Filters.EffectFilters</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="FocusList" name="effectFilters">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="contextMenuPolicy">
             <enum>Qt::CustomContextMenu</enum>
            </property>
           </widget>
          </item>
          <item alignment="Qt::AlignLeft">
           <widget class="QWidget" name="widget_2" native="true">
            <layout class="QHBoxLayout" name="horizontalLayout_6">
             <property name="spacing">
              <number>4</number>
             </property>
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QPushButton" name="addEffectFilter">
               <property name="maximumSize">
                <size>
                 <width>22</width>
                 <height>22</height>
                </size>
               </property>
               <property name="text">
                <string notr="true"/>
               </property>
               <property name="icon">
                <iconset resource="obs.qrc">
                 <normaloff>:/res/images/add.png</normaloff>:/res/images/add.png</iconset>
               </property>
               <property name="autoDefault">
                <bool>false</bool>
               </property>
               <property name="flat">
                <bool>true</bool>
               </property>
               <property name="themeID" stdset="0">
                <string>addIconSmall</string>
               </property>
               <property name="accessibleName">
                <string>Add</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="removeEffectFilter">
               <property name="maximumSize">
                <size>
                 <width>22</width>
                 <height>22</height>
                </size>
               </property>
               <property name="text">
                <string notr="true"/>
               </property>
               <property name="icon">
                <iconset resource="obs.qrc">
                 <normaloff>:/res/images/list_remove.png</normaloff>:/res/images/list_remove.png</iconset>
               </property>
               <property name="autoDefault">
                <bool>false</bool>
               </property>
               <property name="flat">
                <bool>true</bool>
               </property>
               <property name="themeID" stdset="0">
                <string>removeIconSmall</string>
               </property>
               <property name="accessibleName">
                <string>Remove</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="moveEffectFilterUp">
               <property name="maximumSize">
                <size>
                 <width>22</width>
                 <height>22</height>
                </size>
               </property>
               <property name="text">
                <string notr="true"/>
               </property>
               <property name="icon">
                <iconset resource="obs.qrc">
                 <normaloff>:/res/images/up.png</normaloff>:/res/images/up.png</iconset>
               </property>
               <property name="autoDefault">
                <bool>false</bool>
               </property>
               <property name="flat">
                <bool>true</bool>
               </property>
               <property name="themeID" stdset="0">
                <string>upArrowIconSmall</string>
               </property>
               <property name="accessibleName">
                <string>MoveUp</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="moveEffectFilterDown">
               <property name="maximumSize">
                <size>
                 <width>22</width>
                 <height>22</height>
                </size>
               </property>
               <property name="text">
                <string notr="true"/>
               </property>
               <property name="icon">
                <iconset resource="obs.qrc">
                 <normaloff>:/res/images/down.png</normaloff>:/res/images/down.png</iconset>
               </property>
               <property name="autoDefault">
                <bool>false</bool>
               </property>
               <property name="flat">
                <bool>true</bool>
               </property>
               <property name="themeID" stdset="0">
                <string>downArrowIconSmall</string>
               </property>
               <property name="accessibleName">
                <string>MoveDown</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="rightContainerLayout">
       <item>
        <layout class="QVBoxLayout" name="rightLayout">
         <item>
          <widget class="OBSQTDisplay" name="preview" native="true">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>24</width>
             <height>24</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <property name="spacing">
          <number>4</number>
         </property>
         <property name="sizeConstraint">
          <enum>QLayout::SetMaximumSize</enum>
         </property>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QDialogButtonBox" name="buttonBox">
           <property name="standardButtons">
            <set>QDialogButtonBox::Reset|QDialogButtonBox::Close</set>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
  <action name="actionRemoveFilter">
   <property name="icon">
    <iconset resource="obs.qrc">
     <normaloff>:/res/images/list_remove.png</normaloff>:/res/images/list_remove.png</iconset>
   </property>
   <property name="text">
    <string>Remove</string>
   </property>
   <property name="shortcut">
    <string>Del</string>
   </property>
  </action>
  <action name="actionMoveUp">
   <property name="icon">
    <iconset resource="obs.qrc">
     <normaloff>:/res/images/up.png</normaloff>:/res/images/up.png</iconset>
   </property>
   <property name="text">
    <string>Basic.MainMenu.Edit.Order.MoveUp</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Up</string>
   </property>
  </action>
  <action name="actionMoveDown">
   <property name="icon">
    <iconset resource="obs.qrc">
     <normaloff>:/res/images/down.png</normaloff>:/res/images/down.png</iconset>
   </property>
   <property name="text">
    <string>Basic.MainMenu.Edit.Order.MoveDown</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Down</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>OBSQTDisplay</class>
   <extends>QWidget</extends>
   <header>qt-display.hpp</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>FocusList</class>
   <extends>QListWidget</extends>
   <header>focus-list.hpp</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="obs.qrc"/>
 </resources>
 <connections>
 </connections>
</ui>
